From phil@ipom.com Thu Nov  2 23:14:15 2006
Date: Thu, 02 Nov 2006 23:14:10 -0800
From: Phil Dibowitz <phil@ipom.com>
To: Greg KH <greg@kroah.com>, USB Storage list <usb-storage@lists.one-eyed-alien.net>, <alexandre.nunes@gmail.com>
Subject: USB: Fix UCR-61S2B unusual_dev entry
Message-ID: <454AEC42.7010903@ipom.com>


Recently this entry's bcd scope was narrowed so as not to falsly apply
to bcd's other than 0x0110. But while it breaks those of a larger bcd,
it is still needed for those of a smaller bcd - so this changes the
lower bcd limit to 0x0000.

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -1323,8 +1323,10 @@ UNUSUAL_DEV(  0x0fce, 0xe030, 0x0000, 0x
 /* Reported by Kevin Cernekee <kpc-usbdev@gelato.uiuc.edu>
  * Tested on hardware version 1.10.
  * Entry is needed only for the initializer function override.
+ * Devices with bcd > 110 seem to not need it while those
+ * with bcd < 110 appear to need it.
  */
-UNUSUAL_DEV(  0x1019, 0x0c55, 0x0110, 0x0110,
+UNUSUAL_DEV(  0x1019, 0x0c55, 0x0000, 0x0110,
 		"Desknote",
 		"UCR-61S2B",
 		US_SC_DEVICE, US_PR_DEVICE, usb_stor_ucr61s2b_init,
